editor/assets/stylesheets/_z-index.scss

```diff
@@ -13,8 +13,8 @@ $z-layers: (
 	'.editor-post-visibility__dialog': 30,
 	'.editor-post-schedule__dialog': 30,
 	'.editor-block-mover': 30,
-	'.components-notice-list': 100000,
-	'.components-popover': 100000,
+	'.components-notice-list': 9989,
+	'.components-popover': 9989,
```

Comment:
`9989` was the exact value for the notification to fall exactly under the dropdown admin navigation menu, nothing more than that.